src/style.css

```diff
@@ -0,0 +1,145 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgba(128, 128, 128, 0.233);
+  font-family: 'poppins', sans-serif;
+  margin: 50px;
+}
+
+header {
+  font-size: 1rem;
+  width: 100%;
+  height: 3rem;
+  padding: 10px;
+  padding-right: 20px;
+  background-color: rgb(247, 241, 241, 0.425);
+  margin-top: 30px;
+  margin-bottom: 5px;
+  display: flex;
```

Comment:
- [ ] You have used display: flex 5 times, and this falls under to category of Don't repeat yourself. I would suggest you create a class in the CSS with display: flex and give to the classes that you want in HTML.